src/commands/sort_ro.json

```diff
@@ -19,8 +20,7 @@
             {
                 "flags": [
                     "RO",
-                    "ACCESS",
-                    "INCOMPLETE"
```

Comment:
@oranagra I didn't understand this change from your other PR, this isn't the part that is incomplete, it's the next section that is.

Reply:
I think SORT has two issues (each one deserves an INCOMPLETE flag due to a different reason).
it has an input keys key-spec, that's only able to find the key to sort, but is not able to find the keys behind the BY and GET patterns.
and it has an output keys key-spec (which is missing in SORT_RO), that has an INCOMPLETE flag on it's own right (maybe i'm wrong? @guybe7)

either way, i think the first INCOMPLETE, belongs to the input keys key-spec.
the other alternative is to add a 3rd spec for these BY and GET keys.
WDYT?

Reply:
SORT:
1. inpput key: regular "range" spec
2. BY/GET: "incomplete" and "unknown"
3. STORE: "incomplete" and "unknown"

SORT_RO: same as SORT, except the last one